python_modules/dagster-webserver/dagster_webserver_tests/test_smoke.py

```diff
@@ -29,11 +30,13 @@
 )
 def test_smoke_app(gen_instance):
     with gen_instance() as instance:
-        with get_workspace_process_context_from_kwargs(
+        with WorkspaceProcessContext(
             instance,
             version="",
             read_only=False,
-            kwargs=dict(module_name=("dagster_webserver_tests.toy.bar_repo",), definition="bar"),
```

Comment:
`definition` is not even a valid argument. Use of unconstrained `kwargs` led to several instances of "fake" tests, where old arguments continued to be present in the tests and passed into the CLI tests, but weren't actually used.